docs/src/pages/infrastructure/required-services/index.mdx

```diff
@@ -108,40 +108,38 @@ kubectl create configmap kafka-brokers --from-literal=brokers='<replace with com
 kubectl describe configmap kafka-brokers -n <target k8s namespace / ocp project>
 ```
 
-#### Event Streams API Key
+#### Event Streams User Credentials
 
-The Event Streams Broker API Key is needed to connect any deployed consumers or producers to the service running in your cluster. To avoid sharing security keys, create a Kubernetes Secret in the target cluster you will deploy the application microservices to.  You can specify keys at the topic and consumer group levels or use a unique key for all topics and all consumer groups.
+The Event Streams Scram User Credentials are needed in order for any deployed consumers or producers to work with the IBM Event Streams instance running in your cluster. These SCRAM credentials are associated to the KafkaUser object that is being created behind the scenes. In order to create that KafkaUser object and obtain the SCRAM credentials for it, follow the instructions at <https://ibm.github.io/event-streams/security/managing-access/#creating-a-kafkauser-in-the-ibm-event-streams-ui>
```

Comment:
You'll want to remove the <> from around the URL here and just leave it as a plain URL.